source/Preferences.cpp

```diff
@@ -250,6 +255,7 @@ void Preferences::Save()
 	out.Write("Flotsam collection", flotsamIndex);
 	out.Write("view zoom", zoomIndex);
 	out.Write("vsync", vsyncIndex);
+	out.Write("date format", static_cast<int>(Preferences::GetDateFormat()));
```

Comment:
GetDateFormat casts the int index to a DateFormat, and this is casts it back to an int. Why not just use the index variable?
```suggestion
	out.Write("date format", dateFormatIndex);
```